### PR TITLE
make text icons and add to charcuterie

### LINKED ISSUE #8: Text icons
image
Large size ^ 
and a smaller one that's a similar size to text 

Add to charcuterie when done

### PR DESCRIPTION
**Pull request by acchiang**
Actual code: 
image

Design: 
image

Props: 
- textLetter: the letter inside the icon
- children: the name under the icon 
- className: classname for icon
- size: of icon, can be "small" or "default"
- complete: for future expansion, doesn't do anything right now. Could be implemented like a hover or disabled state. eg. when a user represented by the icon is completed ordering, the complete state could show the icon with a different color.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 import Charcuterie from 'pages/Charcuterie';
 import reportWebVitals from './utils/reportWebVitals';
+import TextIcon from 'components/TextIcon'
 
 ReactDOM.render(
   <React.StrictMode>
     {/* <App /> */}
     <Charcuterie >
       <p>Hi</p>
+      <TextIcon textLetter={'a'} size={'small'} color={'#8EDB31'}>small</TextIcon>
```
Comment (christylo) [inline src/index.js:12]: do we wanna use constants for color?

File: src/index.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './pages/App';
+// import App from './pages/App';
```
Comment (kozr) [inline src/index.js]:
```suggestion

```

**Review by kozr — APPROVED** [review]

File: src/index.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 import Charcuterie from 'pages/Charcuterie';
 import reportWebVitals from './utils/reportWebVitals';
+import TextIcon from 'components/TextIcon'
 
 ReactDOM.render(
   <React.StrictMode>
     {/* <App /> */}
```
Comment (kozr) [inline src/index.js]: I just forgot to remove it
```suggestion
```

File: src/index.js (hunk 1 above)
Comment (rzyxie) [reply src/index.js:12]: how would the colours for each user icon get picked (in the future when we start implementing that part)? will it be something like randomly generating from a list of colour constants and then passed into TextIcon?

**Review by rzyxie — APPROVED** [review]
very nice

File: src/index.js (hunk 1 above)
Comment (acchiang) [reply src/index.js:12]: random colors from a set of constants most likely, but since this is charcuterie i'm cool with just a string literal for now

**Pull request merged**